tex/gregoriotex-signs.tex

```diff
@@ -1620,20 +1620,23 @@
 \def\gre@bar@text#1{\ifcase#1 \or text\fi}%
 
 \newcount\gre@count@shiftaftermora
-\gre@count@shiftaftermora=4
+\gre@count@shiftaftermora=5
 
 \def\gresetshiftaftermora#1{%
   \IfStrEq{#1}{always}%
-    {\gre@count@shiftaftermora=4}%
-    {\IfStrEq{#1}{barsonly}%
-      {\gre@count@shiftaftermora=3}%
-      {\IfStrEq{#1}{barsnotextonly}%
-        {\gre@count@shiftaftermora=2}%
-        {\IfStrEq{#1}{barsinsideonly}%
-          {\gre@count@shiftaftermora=1}%
-          {\IfStrEq{#1}{never}%
-            {\gre@count@shiftaftermora=0}%
-            {\gre@error{Unrecognized option "#1" in gresetshiftaftermora , use 'always', 'barsonly', 'barsnotextonly', 'barsinsideonly' or 'never'}}%
+    {\gre@count@shiftaftermora=5}%
```

Comment:
you're right... but the clock is ticking, we'll clean this in the future (maybe)